docs/src/overview.md

```diff
@@ -50,7 +49,7 @@ SciML workflows.
 | Fast Fourier Transform                                                                                  | [FFTW\*](https://github.com/JuliaMath/FFTW.jl)                                                                                                                          |
 | Partial Differential Equation Discretizations                                                           | Associated Julia packages                                                                                                                                               |
 | ---                                                                                                     | ---                                                                                                                                                                     |
-| Finite Differences                                                                                      | [MethodOfLines](https://docs.sciml.ai/MethodOfLines/stable/)                                                                                                                    |
+| Finite Differences                                                                                      | [MethodOfLines](https://docs.sciml.ai/MethodOfLines/stable/), [FiniteDifferences\*](https://juliadiff.org/FiniteDifferences.jl/latest/), [FiniteDiff\*](https://github.com/JuliaDiff/FiniteDiff.jl) |
```

Comment:
Those two are different, they aren't PDE discretizers.

Reply:
Okay, well it's not clear to me what the break at line 51 is indicating, or why there are two separate rows with the "Workflow Element" "Finite Differences".